src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -694,7 +518,7 @@
               Visibility="{x:Bind local:GraphingCalculator.ShouldDisplayPanel(IsSmallState, SwitchModeToggleButton.IsChecked.Value, x:False), Mode=OneWay}">
             <Grid.RowDefinitions>
                 <RowDefinition Height="*"/>
-                <RowDefinition Height="1.8*"/>
+                <RowDefinition Height="2.2*" MaxHeight="400"/>
```

Comment:
@grochocki  & @sanderl  also mentioned that they want the right pane to max out at a max width of around ~400 also. Since you are already adding this here can we restrict the width also?

Reply:
done